apps/shooting-clubs/acceptance/features/storage-address.js

```diff
@@ -0,0 +1,187 @@
+'use strict';
+
+const steps = require('../../');
+
+Feature('Shooting club Storage Address step');
+
+Before((
+  I,
+  storageAddressPage
+) => {
+  I.visitPage(storageAddressPage, steps);
+});
+
+Scenario('The correct form elements are present on storage-postcode step', (
+  I,
+  storageAddressPage
+) => {
+  I.seeElements([
```

Comment:
this accepts a string or an array, I'd just pass the single item if there's only one. Or you could use the build in method seeElement - up to you!